radio_beam/beam.py

```diff
@@ -522,3 +535,84 @@ def __init__(self, width, height, position_angle, support_scaling=8, **kwargs):
                                                       np.max([width,height]))
         super(EllipticalGaussian2DKernel, self).__init__(**kwargs)
         self._truncation = np.abs(1. - 1 / self._array.sum())
+
+
+class EllipticalDisk2D(Fittable2DModel):
+    """
+    Two dimensional elliptical Disk model.
+    Parameters
+    ----------
+    amplitude : float
+        Value of the disk function
+    x_0 : float
+        x position center of the disk
+    y_0 : float
+        y position center of the disk
+    a : float
+        Major axis of the disk
+    b : float
+        Minor axis of the disk
+    theta : float
+        Orientation of the major axis with respect to the x axis.
+
+    See Also
+    --------
+    Box2D, TrapezoidDisk2D, Disk2D
+
+    Notes
+    -----
+    ** NOT UPDATED **
```

Comment:
Ah. Forgot to change that. I copied ```Disk2D``` from astropy and haven't updated to the elliptical model.